Release pipeline failed

### Prerequisites

- [x] Write a descriptive title.
- [x] Make sure you are able to reproduce it on the latest released version
- [x] Search the existing issues.

### Steps to reproduce

Merge to main

### Expected behavior

```console
release pipeline runs
```

### Actual behavior

```console
Release pipeline errors at Amend release commit task
```

### Error details

```console
1
Run git add src/PSSAToJunit.psd1 docs/
5
  
5
markdownlint-cli2 v0.21.0 (markdownlint v0.40.0)
6
Finding: **/*.md !**/.claude/** !**/.overstory/** !**/CHANGELOG.md !**/TODO.md !**/node_modules/**/*
7
Linting: 8 file(s)
8
Summary: 0 error(s)
9
 1/19 changelog.config.js 938.48ms
10
 2/19 CHANGELOG.md 88.74ms
11
 3/19 CODE_OF_CONDUCT.md 7.50ms
12
 4/19 CONTRIBUTING.md 15.70ms
13
 5/19 docs/about_PSSAToJunit.md 2.77ms
14
 6/19 docs/ConvertTo-PSSAJunitXml.md 13.63ms X
15
docs/ConvertTo-PSSAJunitXml.md:52:10 - Unknown word (proga)
16
 7/19 docs/Export-PSSAJunitXml.md 10.09ms X
17
docs/Export-PSSAJunitXml.md:83:10 - Unknown word (proga)
18
 8/19 docs/PSSAToJunit.md 2.57ms
19
 9/19 LICENSE 4.10ms
20
10/19 README.md 7.51ms
21
11/19 src/private/ConvertTo-JunitXml.ps1 8.84ms
22
12/19 src/private/Get-ParentXml.ps1 2.64ms
23
13/19 src/private/Get-TestCaseXml.ps1 3.08ms
24
14/19 src/private/Get-TestSuite.ps1 3.36ms
25
15/19 src/private/Get-TestSuiteXml.ps1 1.85ms
26
16/19 src/PSSAToJunit.psd1 9.50ms
27
17/19 src/PSSAToJunit.psm1 2.44ms
28
18/19 src/public/ConvertTo-PSSAJunitXml.ps1 3.48ms
29
19/19 src/public/Export-PSSAJunitXml.ps1 2.07ms
30
CSpell: Files checked: 19, Issues found: 2 in 2 files.
31
husky - pre-commit script failed (code 1)
32
Error: Process completed with exit code 1.
```

### Environment data

```powershell
NA
```

--- a/.cspell.json
+++ b/.cspell.json
@@ -63,6 +63,7 @@
         "markdownlint",
         "nuspec",
         "platy",
+        "proga",
         "platyps",
         "privatefunc",
         "pssa",

